### ISSUE TITLE
Typeface/font that is more LCARSish

### ISSUE THREAD

**Issue by jrwarwick** [joernweissenborn/lcars#25]
While the current text font is sans serif and pretty clean, I think it is not narrow enough, and some of more recognizable glyphs (like S and C and R and 9) just don't quite feel LCARSish.  I recommend a search on the  [Open font library](https://fontlibrary.org/) first. There are also a few nice commercial fonts that are specifically emulating LCARS, but I imagine we would not be allowed to redistribute.
Also recommend a few fall back options when/if the a CDN source is selected, and maybe a note on how to switch from local to CDN and back.

**Comment by joernweissenborn** [comment]
Yeah, the current font is not the best. We can't use commercial ones, we have to keep our head downs so we will not get a DMCA or things. 

I basically want to leave the font to users, but a nicer default would be cool. But it must be open source and widely available, I don't want to distribute a font with this package or have anywhere a default CDN. The library MUST NOT load anything from somewhere else. It is really restricted to be a small self contained CSS, with audio as notable exception, but users EXCPLICT define where to load the resources.

**Comment by physnoct** [comment]
I think this one is the official font, but it doesn't work well.
https://github.com/wrstone/fonts-startrek/blob/master/Swiss%20911%20Ultra%20Compressed%20BT.ttf

That one is similar but works better.
Swiss911 XCm BT.ttf

**Comment by jrwarwick** [comment]
Good find. Thanks, psynoct. Maybe what we can do is have a supplemental
resources suggestions section where we give the exact font names like these
that are known to be more authentic and/or work better, along with sound
effects download resources.

  --Justin

**Comment by dthv** [comment]
I see the current default is Arial? Arial Narrow works very good for an LCARS Font; many won't see it's not the original. Take Liberation Sans Narrow for a free alternative that you probably can include (haven't checked license, though).

**Comment by jrwarwick** [comment]
Thanks, @dthv . It looks like the situation with Liberation Sans Narrow is: it can be had under the SIL OpenFont License, however it depends on which version! https://github.com/liberationfonts/liberation-sans-narrow#current-release
I also found a pretty nice OpenFonts Library option: Bebas Neue Regular, though it is capitals-only (which some may consider a feature, not a flaw).

Thanks to the way CSS will automatically fall back gracefully if you list font names that are not actually available to the browser, maybe we could propose a list of fonts in descending best-match order, but we won't include any actual active URL references to the font files, but instead just leave a CSS comment with a couple of known "safe" places to download.

To recap and summarize: we could specify (but not supply) the following open/libre-licensed fonts:
  Liberation Sans Narrow  (https://github.com/liberationfonts/liberation-sans-narrow#current-release)
  Bebas Neue Regular  (https://fontlibrary.org/en/font/bebasneueregular)

And suggest or mention, but not specify or supply:
  Swiss911-XCm-BT-Regular  (https://fontsgeek.com/fonts/Swiss911-XCm-BT-Regular , https://www.myfonts.com/search/swiss+911/fonts/ )

and then it is only up to the user to uncomment a corresponding font-family source element and either modify the local-relative path spec to replace it with a reference to a CDN or to download it to local on their own.

**Comment by dthv** [comment]
> It looks like the situation with Liberation Sans Narrow is: it can be had under the SIL OpenFont License, however it depends on which version! https://github.com/liberationfonts/liberation-sans-narrow#current-release

Actually, Liberation Sans Narrow cannot be had under the SIL OFL, at all. [As of Fedora recommendation](https://fedoraproject.org/wiki/Features/Liberation_Fonts_2#Benefit_to_Fedora), most of the *Liberation* fonts are (now) licensed under SIL OFL. However, due to some unspecified problem, *Liberation Sans Narrow* is not. Instead, it is still (as the other fonts were) licensed [under a modified GPL license](https://github.com/liberationfonts/liberation-sans-narrow/blob/master/License.txt).

I don't know whether the GPL is compatible with lcars' MIT license, though. But since the maintainer doesn't want to include fonts, that's not important, after all.

In any case: The Liberation font family used to be pretty widespread among several Linux desktops. I think it's the default font LibreOffice uses on some desktops. So one might specify *Liberation Sans Narrow* as well as *Arial Narrow* without shipping one of them.

[Antonio](https://www.fontsquirrel.com/fonts/antonio) could also be used, but restricted to uppercase-only (e.g. headlines and/or elements). The lowercase types don't look very LCARSish. If SIL OFL is compatible with lcars' MIT license, Antonio could (also) be included into the project, as could be Bebas Neue Regular (yet, I accept that this is not wanted).

I'd propose the following list:
Liberation Sans Narrow, Bebas Neue Regular, Antonio, Arial Narrow, Arial

Swiss911-XCm-BT-Regular should be on a separate list of mentioned fonts, as it's probably too esoteric to expect many uses to have it installed.
